test/inductor/test_torchinductor_opinfo.py

```diff
@@ -250,7 +250,8 @@ def process(device_type):
     "stft": {f32, f64},
     "tensor_split": {b8, f16, f32, f64, i32, i64},
     "to_sparse": {f32, f64},
-    "uniform": {f16, f32, f64},
+    # AssertionError: Tensor-likes are not close!
+    "uniform": {f16},
```

Comment:
Interesting that these started working.  I'd expect randomness to not work with opinfo testing.

Reply:
I'm not sure why eager and inductor randomness match -- not sure if the default generator is global or configured separately and the backend is setting the seed (by the default generator) . 

Here's a simple reproducer:
```
import torch
import torch._dynamo as dynamo

op = torch.Tensor.bernoulli_
args = torch.empty(10000)

def foo(args):
    #torch.manual_seed(42)
    return op(args)

opt_foo = dynamo.optimize("inductor")(foo)

y = foo(args)
y_ = opt_foo(args)
print(y)
print(y_)
print(torch.allclose(y, y_))
```

Reply:
Is there a graph break here?  Try changing `dynamo.optimize("inductor", nopython=True)`.

Reply:
No, there's no graph break with this PR. I could reproduce the randomness match with `nopython=True` as well. I've generally noticed they match for CPU fp32, fp64.